### PR TITLE
Attempt to add genomics repos

### PR DESCRIPTION
**Pull request by ErinBecker**
I'm trying to add the repos for the genomics lessons to mirror structure for ecology lessons and make it easier for instructors to find additional materials (like instructor notes) that are only present in repos and not on website.

Someone please check my html and make sure it works. . .

### CODE REVIEW

**Comment by ErinBecker** [comment]
@tracykteal Could you check to make sure my html looks ok and merge?

File: pages/lessons.md [hunk 1]
```diff
@@ -92,37 +92,44 @@ The focus of this workshop is on working with genomics data and data management
...
   <tr>
     <td>Introduction to the workshop and dataset</td>
     <td><a href="{{site.dc_website_url}}/introduction-genomics/" target="_blank" class="icon-browser" title="icon-browser"></a></td>
+    <td><a href="{{site.dc_github_repo_url}}/introduction-genomics/" target="_blank" class="icon-github" title="icon-github"></a></td>
     <td>Tracy Teal</td>
   </tr>
   <tr>
     <td>Introduction to cloud computing for genomics</td>
     <td><a href="{{site.dc_github_repo_url}}/cloud-genomics/tree/gh-pages/lessons" target="_blank" class="icon-browser" title="icon-browser"></a></td>
+   <td><a href="{{site.dc_github_repo_url}}/cloud-genomics-genomics/" target="_blank" class="icon-github" title="icon-github"></a></td>
```
Comment (gvwilson) [inline pages/lessons.md]: Should be 'cloud-genomics', not 'cloud-genomics-genomics'

File: pages/lessons.md [hunk 2]
```diff
@@ -92,37 +92,44 @@ The focus of this workshop is on working with genomics data and data management
...
     <td>Bob Freeman</td>
   </tr>
   <tr>
     <td>Introduction to the command line</td>
     <td><a href="{{site.dc_github_repo_url}}/shell-genomics/tree/gh-pages/lessons" target="_blank" class="icon-browser" title="icon-browser"></a></td>
```
Comment (gvwilson) [inline pages/lessons.md]: This link goes to a GitHub site, not to the rendered page. (The error is in the original file.)

File: pages/lessons.md [hunk 3]
```diff
@@ -92,37 +92,44 @@ The focus of this workshop is on working with genomics data and data management
...
+   <td><a href="{{site.dc_github_repo_url}}/shell-genomics/" target="_blank" class="icon-github" title="icon-github"></a></td>
     <td>Sheldon McKay, Karen Cranston</td>
   </tr>
   <tr>
     <td>Data wrangling and processing</td>
     <td><a href="{{site.dc_github_repo_url}}/wrangling-genomics/tree/gh-pages/lessons" target="_blank" class="icon-browser" title="icon-browser"></a></td>
```
Comment (gvwilson) [inline pages/lessons.md]: This link goes to a GitHub site, not to the rendered page. (The error is in the original file.)

**Pull request merged**